src/licensedcode/data/licenses/hessla.yml

```
@@ -0,0 +1,8 @@
+key: hessla
+short_name: HESSLA 
+name: Hacktivismo Enhanced-Source Software License Agreement
+category: Proprietary
+owner: Hacktivismo
+homepage_url: http://www.hacktivismo.com/about/hessla.php
+text_urls: https://directory.fsf.org/wiki/License:HESSLA
```

This should be a list not a string:
```suggestion
text_urls:
    - https://directory.fsf.org/wiki/License:HESSLA
```